modules/tracker/tests/controllers/ApiControllerTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/*=========================================================================
+ Midas Server
+ Copyright Kitware SAS, 26 rue Louis Guérin, 69100 Villeurbanne, France.
+ All rights reserved.
+ For more information visit http://www.kitware.com/.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0.txt
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+=========================================================================*/
+
+require_once BASE_PATH.'/modules/api/tests/controllers/CallMethodsTestCase.php';
+
+/** Api Test for tracker module */
+class Tracker_ApiControllerTest extends Api_CallMethodsTestCase
+{
+    public $moduleName = 'tracker';
+
+    /** Setup. */
+    public function setUp()
+    {
+        $this->enabledModules = array('api', 'scheduler', $this->moduleName);
+        $this->_models = array('Assetstore', 'Community', 'Setting', 'User');
+
+        $this->setupDatabase(array('default'));
+        $this->setupDatabase(array('default'), $this->moduleName);
+
+        ControllerTestCase::setUp();
+    }
+
+    /** Test the AJAX get free space call. */
+    public function testUploadScalarWithSubmission()
+    {
+        $uuid = uniqid();
```

Comment:
Use [core/controllers/components/UuidComponent.php](https://github.com/midasplatform/Midas/blob/master/core/controllers/components/UuidComponent.php) and anywhere else `uniqid()` is used, if any.